GNUmakefile

```diff
@@ -90,4 +90,13 @@ lint:
 update_release_schema:
 	go run helpers/schema-api/main.go -export .release/provider-schema.json
 
+tflint:
+	tflint --recursive -f compact --config="$(CURDIR)/.tflint.hcl" --fix
+
+tflint_check:
+	tflint --recursive -f compact --config="$(CURDIR)/.tflint.hcl"
+
+tf_fmt:
+	terraform fmt -recursive
+
```

Comment:
```
tflint_fix_examples:
 # fix examples linting
tffmt_examples:
 # format examples
check_examples:
 # linting + formatting checks on examples directory
```

Reply:
may be shorter?
- tflint_fix
- tffmt
- check_examples

If you want to do formatting, then it takes a long time to type such commands. And we have terraform files anyway only in examples

Reply:
Setup bash autocompletion. :)